docs/tools/buildkit.md

```diff
@@ -135,6 +108,84 @@ If you want to ensure that the buildkit CLI tools are always available, then:
     Each time you open a new terminal while working on Civi development, you would need to re-run the `export` command.
 
 
+### Configuring `amp` {:#amp-config}
+
+Buildkit provides a tool called `amp` which [civibuild](/tools/civibuild.md) uses when it needs to set up a new site. Before you can use `civibuild`, need to configure `amp` by telling it a bit about your system (e.g. what webserver you're using). 
+
+#### Interactive config
+
+```
+$ amp config
+```
+
+!!! tip "tips"
+    * Run this as a non-`root` user who has `sudo` permission. This will ensure that new files are owned by a regular user, and (if necessary) it enables `civibuild` to restart Apache or edit `/etc/hosts`.
+    * Pay close attention to any instructions given in the output of this command.  They may involve adding a line to your Apache configuration file.
```

Comment:
Should also mention nginx perhaps change to "Pay close attention to any instructions given in the output of this command. They may involving adding a line to your web-server configuration (apache or nginx)